com.moura.Progress.json

```diff
@@ -0,0 +1,153 @@
+                	"/lib/pkgconfig"
+            	    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://www.cairographics.org/releases/cairomm-1.16.2.tar.xz",
+                            "sha512": "61dc639eabe8502e1262c53c92fe57c5647e5ab9931f86ed51e657df1b7d0e3e58c2571910a05236cc0dca8d52f1f693aed99a553430f14d0fb87be1832a6b62"
+                        }
+                    ]
+                },
+                {
+                    "name": "glibmm",
+                    "buildsystem": "meson",
+                    "config-opts": [
+                        "-Dbuild-examples=false"
+                    ],
+                    "cleanup": [
+		        "*.la",
+		        "/include/glibmm-2.68",
+		        "/include/giomm-2.68",
+		        "/lib/glibmm-2.68",
+		        "/lib/giomm-2.68",
+		        "/lib/libglibmm_generate_extra_defs*",
+		        "/lib/pkgconfig"
+            	    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/glibmm/2.68/glibmm-2.68.2.tar.xz",
+                            "sha256": "91e0a8618f7b82db4aaf2648932ea2bcfe626ad030068c18fa2d106fd838d8ad"
+                        }
+                    ]
+                },
+                {
+                    "name": "pangomm",
+                    "buildsystem": "meson",
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/pangomm/2.50/pangomm-2.50.2.tar.xz",
+                            "sha256": "1bc5ab4ea3280442580d68318226dab36ceedfc3288f9d83711cf7cfab50a9fb"
+                        }
+                    ],
+                    "cleanup": [
+		        "*.la",
+		        "/include/pangomm-2.48",
+		        "/lib/pangomm-2.48",
+		        "/lib/pkgconfig"
+            	    ]
+                }
+            ]
+        },
+        {
+            "name": "tinyxml2-10",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/leethomason/tinyxml2.git",
+                    "tag": "10.0.0"
+                }
+            ]
+        },
+        {
+            "name": "progress-tracker",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DFLATPAK=True",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/smolBlackCat/progress-tracker.git",
+                    "tag": "v1.0"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--filesystem=home",
```

Comment:
is home directory access really needed?

Reply:
Preferably yes, as I don't expect the users to have their image files located specifically at xdg-pictures. Considering as well that the app saves the boards as .xml files in the .config directory

Reply:
it shouldn't use `.config` and instead follow XDG directories standard.